vnfs/bin/wwmkchroot.in

```diff
@@ -38,23 +40,34 @@ usage() {
     echo
 }
 
+unset VERBOSE
+unset DEBUG
+unset NOGPGCHECK
+DEVNULL="/dev/null"
 
 ### Argument processing
-while getopts "a:dhv" opt; do
+while getopts ":dhvg" opt; do
     case $opt in
         d)
             VERBOSE=1
+            DEBUG=1
+            DEVNULL="/dev/stdout"
             set -x
         ;;
         v)
             VERBOSE=1
         ;;
+       \?)
+            printf "ERROR: $OPTARG not recognized.\n\n" 1>&2
+            usage
+            exit 1
+        ;;
         h)
             usage
             exit 0
         ;;
-        a)
```

Comment:
This can probably be removed. I don't think it wasn't used much. IIRC, we had this as a way for simple passing of a value into the `*.tmpl` file...  So, `$var` was used in the template, and you'd do:

```
wwmkchroot -a bob [...]
```

To pass the `bob` value in.  Hackish...

Reply:
I've seen it used. Close this comment?

Reply:
I meant "never seen -a used".